kafka/pykafka/simpleconsumer.py

```diff
@@ -102,21 +85,26 @@ def __init__(self,
             if no message is available for consumption after the specified interval
         :type consumer_timeout_ms: int
         """
-        self._cluster = cluster
+        if not isinstance(cluster, weakref.ProxyType):
+            self._cluster = weakref.proxy(cluster)
+        else:
+            self._cluster = cluster
         self._consumer_group = consumer_group
         self._topic = topic
         self._fetch_message_max_bytes = fetch_message_max_bytes
-        self._socket_timeout_ms = socket_timeout_ms
         self._fetch_min_bytes = fetch_min_bytes
+        self._queued_max_messages = queued_max_messages
+        self._num_consumer_fetchers = num_consumer_fetchers
+        self._fetch_wait_max_ms = fetch_wait_max_ms
+        self._consumer_timeout_ms = consumer_timeout_ms
 
         self._last_message_time = time.time()
-        self._consumer_timeout_ms = consumer_timeout_ms
 
         self._auto_commit_enable = auto_commit_enable
         self._auto_commit_interval_ms = auto_commit_interval_ms
         self._last_auto_commit = time.time()
 
-        self._queued_max_messages = queued_max_messages
+        self._running = True
```

Comment:
Small thing, but you might want to set this after the worker thread has started.